config.go

```diff
@@ -1045,15 +1056,27 @@ func loadConfig() (*config, error) {
 			"minbackoff")
 	}
 
-	// Validate the subconfigs for workers and caches.
+	// Validate the subconfigs for workers, caches, and the tower client.
 	err = lncfg.Validate(
 		cfg.Workers,
 		cfg.Caches,
+		cfg.WtClient,
 	)
 	if err != nil {
 		return nil, err
 	}
 
+	// If the user provided private watchtower addresses, parse them to
+	// obtain the LN addresses.
+	if cfg.WtClient.IsActive() {
```

Comment:
Just noting here that we'll want to start the client regardless of whether any addresses are given since we'll be able to add them through the RPC sub-server. Can either be done now or in the PR introducing it.

Reply:
would vote for addressing it in the subsequent PR, since we will probably need to modify some interfaces to handle the fact that activity is mutable